tests/simulation/physics/test_sparams.py

```
@@ -0,0 +1,204 @@
+"""Physics simulation tests: calculate_sparam for a simple waveguide.
+
+Tests that a lossless Si/SiO2 slab waveguide transmits ~100% of
+fundamental-TE-mode light, measured at multiple ModeOverlapDetectors
+at increasing distances from the source.
+
+Geometry (propagation along x, periodic in y = infinite slab approximation):
+  Domain:  3 µm × 150 nm × 2 µm (including PML)
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)